tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py26,py27,py33,py34
+envlist = py27,py34,py35
```

Comment:
Also need to change Travis.  I've found `python: 3.5` is the most succinct way to ensure py35 is available